src/value/__tests__/use-transform.test.tsx

```diff
@@ -53,12 +73,15 @@ describe("as input/output range", () => {
             const x = useMotionValue(100)
             const opacity = useTransform(x, [0, 200], [0, 1])
 
-            x.set(20)
+            React.useEffect(() => {
```

Comment:
Subscriptions are now added via useEffect, which is the recommended concurrent-safe way to establish subscriptions in React. So these tests have been updated to reflect that.